rsocket-core/src/test/java/io/rsocket/test/util/LocalSocketAddress.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2015-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.rsocket.test.util;
+
+import java.net.SocketAddress;
+import java.util.Objects;
+
+public final class LocalSocketAddress extends SocketAddress {
```

Comment:
Suggest putting Test in the name of the class maybe toString to make it non-confusing with https://github.com/rsocket/rsocket-java/blob/d903e9635a159285b6943ea93156c31aa406ba5d/rsocket-transport-local/src/main/java/io/rsocket/transport/local/LocalSocketAddress.java